### PR TITLE
erlang-native: Add missing dependency

### PR DESCRIPTION
**Pull request by chbae**
[ERROR]
configure: error: No curses library functions found

Signed-off-by: Changhyeok Bae <changhyeok.bae@gmail.com>

### CODE REVIEW

File: recipes-devtools/erlang/erlang-native_18.1.5.bb [hunk 1]
```diff
@@ -5,6 +5,8 @@ inherit native
 
 PR = "r0"
 
+DEPENDS_append = " ncurses-native"
```
Comment (bboozzoo) [inline recipes-devtools/erlang/erlang-native_18.1.5.bb]: I think you could replace adding this line to every `erlang-native-*` native recipe with the following line:
```
DEPENDS_append_class-native = " ncurses-native"
```
added to `erlang.inc`

Comment (chbae) [inline recipes-devtools/erlang/erlang-native_18.1.5.bb]: That's great :) I'll do it. Thanks.

**Comment by bboozzoo** [comment]
Looks good to me.

**Comment by chbae** [comment]
This is also applied into krogoth branch.

**Comment by joaohf** [comment]
Done.

Thanks

**Pull request merged**